spec/index.md

```diff
@@ -5033,6 +5057,10 @@ S with
 
 ```
 
+#### IC Management Canister: Canister logs
+
+TBD
```

Comment:
also changes to do the abstract IC state and system API specifications are missing yet (and thus TBD)

Reply:
I added the abstract IC state description.
but I'm not sure what other system API specifications are missing, could you please clarify?

Reply:
I've refined your initial spec PR so let's continue the discussion on the detailed questions I raised below.